src/odemis/acq/stitching/_weaver.py

```diff
@@ -43,66 +44,233 @@ def collageWeaver(tiles, border=None):
     return (2D DataArray): same dtype as the tiles, with shape corresponding to
       the bounding box.
     """
+    
+    def __init__(self):
+        self.tiles = []
+
+    def addTile(self,tile):
+        """
+        tile (2D DataArray): the image must have at least MD_POS and
+        MD_PIXEL_SIZE metadata. All provided tiles should have the same dtype.
+        """
+        
+        self.tiles.append(tile)
+    
```

Comment:
No need for too many blank lines. No blank line at the beginning of the method, and just one at the end (before the next method).